src/mpi/romio/adio/ad_daos/ad_daos_common.c

```diff
@@ -1,22 +1,20 @@
 /* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
 /*
+ *  (C) 2020 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
  *
- * Copyright (C) 2018-2020 Intel Corporation
- *
- * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
- * The Government's rights to use, modify, reproduce, release, perform, display,
- * or disclose this software are subject to the terms of the Apache License as
- * provided in Contract No. 8F-30005.
- * Any reproduction of computer software, computer software documentation, or
- * portions thereof marked with this legend must also reproduce the markings.
+ *  Portions of this code were written by Intel Corporation.
+ *  Copyright (C) 2018-2020 Intel Corporation.  Intel provides this material
+ *  to Argonne National Laboratory subject to Software Grant and Corporate
+ *  Contributor License Agreement dated February 8, 2012.
```

Comment:
All of the contribution copyright material is being moved to the top-level copyright.  Specific CLA related information (such as the one above) should be maintained in the PR, not in the files.

Reply:
hmm those are already in your master.
I guess you are not looking at the squashed changes. maybe i can squash this commit if that helps.

Reply:
No, the move is not in master yet.  It's in a PR waiting for confirmation from our legal team.